site/using-smart-accounts/export-private-key.md

```diff
@@ -0,0 +1,35 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Alchemy Signer • Export Private Key
+  - - meta
+    - name: description
+      content: Learn how to enable a user to export their private key with the Alchemy Signer
+  - - meta
+    - property: og:description
+      content: Learn how to enable a user to export their private key with the Alchemy Signer
+  - - meta
+    - name: twitter:title
+      content: Alchemy Signer • Export Private Key
+  - - meta
+    - name: twitter:description
+      content: Learn how to enable a user to export their private key with the Alchemy Signer
+---
+
+# Export private key
+
+The Alchemy Signer allows you to export a user's private key, allowing them a right to exit at any time. It is considered a best practice to allow your users to export their private key, as it gives them full control over their account. The private key export method does not rely on Alchemy's infrastructure, so even if Alchemy is down, a user can still export their private key.
```

Comment:
for later: we should consider having a status page for our signer apis to link to here for teams to see if we're down